model/database/registry.yaml

```diff
@@ -5,23 +5,31 @@ groups:
     brief: >
       This group defines the attributes used to describe telemetry in the context of databases.
     attributes:
-      - id: db.collection.name
+      - id: db.target.name
         type: string
         stability: experimental  # RELEASE CANDIDATE
-        brief: The name of a collection (table, container) within the database.
+        brief: >
+          The name of the operation target such as collection, table, container,
+          stored procedure, database, or another database object.
         note: |
-          It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization.
+          For data manipulation operations, target usually matches database table,
+          view, stored procedure or a user-defined function name.
+          For data definition or administrative operations, target would match the
+          database, index, user, or some other object type.
+
+          It is RECOMMENDED to capture the value as provided by the application
+          without attempting to do any case normalization.
 
-          A single database query may involve multiple collections.
+          A single database query may involve multiple targets.
```

Comment:
Understand, and what should you do then? I think this statement is not useful, keep only line 32.

Reply:
makes sense, updated the wording